adoc/CaaSP_DH2X_Install_Guide.adoc

```diff
@@ -276,101 +296,109 @@ $ helm init --client-only
 ----
 
 
-==== Optional Components Deployed in the SUSE CaaS Platform 3 Cluster
+==== Deploying Optional Components in the SUSE CaaS Platform 3 Cluster
 
-You can also deploy some useful optional applications in your SUSE CaaS Platform 3 cluster:
+You can deploy some useful optional applications in your SUSE CaaS Platform 3 cluster:
 
-* Heapster (creates simple graphics)
+* Heapster allows analyzing cluster performance
 
-* Kubernetes dashboard (allows administration and basic monitoring of the Kubernetes cluster)
+* The Kubernetes dashboard allows administrating and basic monitoring of the Kubernetes cluster
 
+See also https://documentation.suse.com/suse-caasp/3/html/caasp-admin/.
 
-See also https://www.suse.com/documentation/suse-caasp-3/book_caasp_admin/data/book_caasp_admin.html
-
-Installation of Heapster:
-
+. Install Heapster:
++
 ----
 $ helm install --name heapster-default --namespace=kube-system stable/heapster \
 --version=0.2.7 --set rbac.create=true
 ----
 
-Installation of the Kubernetes dashboard:
-
+. Install the Kubernetes dashboard:
++
 ----
 $ helm install --namespace=kube-system \
 --name=kubernetes-dashboard stable/kubernetes-dashboard \
 --version=0.6.1
 ----
 
-To authenticate against the Kubernetes cluster extract the *id-token* from the *kubeconfig* file.
+. Start the Kubernetes API proxy:
++
 ----
-$ grep id-token $KUBECONFIG | awk '{ print $2 }'
+$ kubectl proxy
 ----
 
+. Extract the *id-token* from the *kubeconfig* file:
++
 ----
-$ kubectl proxy
+$ grep id-token $KUBECONFIG | awk '{ print $2 }'
 ----
++
+This token will allow you to log in in the next step.
 
-Point your browser to
+. In your browser, open:
++
 ----
 http://127.0.0.1:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/
 ----
-
-Use the ID grep'ed from *kubeconfig* to log in to the Kubernetes dashboard.
++
+Use the ID you found in *kubeconfig* to log in to the Kubernetes dashboard.
 
 
 
-== Installation of SAP Data Hub 2
+== Installing SAP Data Hub 2
 
 The following sections describe the preparation and installation of SAP Data Hub 2 on the SUSE CaaS Platform 3 Cluster.
 
 
-=== Prepare the SAP Data Hub 2 Installation
+=== Preparing the SAP Data Hub 2 Installation
 
 To install SAP Data Hub on SUSE CaaS Platform successfully, perform the actions detailed below.
 
 
-==== Download the SAP Data Hub 2 Software Archive
+==== Downloading the SAP Data Hub 2 Software Archive
 
-Go to the SAP Software Download Center, log in with your SAP account and search for SAP DATA HUB 2 or access this link.
-Download the SAP Data Hub Foundation file, for example: DHFOUNDATION03_3-80004015.ZIP (SAP DATA HUB - FOUNDATION 2.3) or DHFOUNDATION06_1-80004015.ZIP (SAP DATA HUB - FOUNDATION 2.6)
+. Go to the SAP Software Download Center, log in with your SAP account and search for `SAP DATA HUB 2` or access this link.
 
+// FIXME: Where is the link mentioned above?
```

Comment:
Fixme for Ulrich

Reply:
???

Reply:
Context above is "Go to the SAP Software Download Center, log in with your SAP account and search for `SAP DATA HUB 2` or access this link."

Reply:
https://launchpad.support.sap.com/#/softwarecenter/template/products/_APP=00200682500000001943&_EVENT=NEXT&HEADER=Y&FUNCTIONBAR=Y&EVENT=TREE&NE=NAVIGATE&ENR=73555000100800000791&V=MAINT&TA=ACTUAL/SAP%20DATA%20HUB

that is a not real nice link ....
Let's try it this way:
Log in to the SAP launchpad, select software download on premise
https://launchpad.support.sap.com/#/softwarecenter
Select "Installations and Upgrades" by alphabetical order, Select "D" and scroll down in the list to folder SAP Data Hub, open this folder and select the version to be installed.

Reply:
No, that is indeed not a nice URL (and trying to shorten by removing parameters only had me staring at a "Loading..." screen). Let's take the instruction then, thanks!